cachi2/core/package_managers/yarn/project.py

```diff
@@ -72,6 +72,12 @@ def registry_server_for_scope(self, scope: str) -> str:
 
         return registry or self.registry_server
 
+    def to_file(self) -> None:
+        """Write the data to the yarnrc file."""
+        with self._path.path.open("w") as f:
+            yaml.safe_dump(self._data, f)
+            f.write("\n")
```

Comment:
But trailing whitespace characters are usually dropped/checked with automation anyway, so why do we want to add it? I mean it's still going to be a valid (autogenerated) YAML which Yarn's YAML parser should handle just fine.

Reply:
Well, fair point. Since this is supposed to be a transparent change to the end user, I guess there's no point in adding the trailing whitespace. Removed it.